message XStreamPack
{
    required bytes pack_data = 1;
    optional bytes pack_meta = 2;
}
